config.json

```diff
@@ -151,13 +151,31 @@
             ]
         },
         {
-            "name": "Migrating to Newer SDL iOS Versions",
+            "name": "Migrating to Newer SDL Versions",
             "navigation": [
                 {
-                    "name": "Updating to v4.3+"
+                    "name": "Updating to v4.3+",
+                    "platforms": ["iOS"]
+                },
+                {
+                    "name": "Updating to v5.0+",
+                    "platforms": ["iOS"]
+                },
+                {
+                    "name": "Updating to 4.4",
+                    "platforms": ["Android","JavaSE","JavaEE"]
+                },
+                {
+                    "name": "Updating to 4.5",
+                    "platforms": ["Android","JavaSE","JavaEE"]
```

Comment:
Remove JavaSE & JavaEE, they were added as available platform in Java_Suite v4.8.